apps/src/templates/sectionProgress/progressTables/ProgressTableView.jsx

```diff
@@ -210,6 +210,10 @@ class ProgressTableView extends React.Component {
   syncScrollTop() {
     clearTimeout(this.timeout);
     this.timeout = setTimeout(() => {
+      if (!this.contentView || !this.studentList) {
```

Comment:
This looks unrelated to the amplitude fix in the rest of this PR.  Is it related?  OR is this seeking to solve another problem?  I don't think it fixes the null pointer exception...

Reply:
Approving, assuming this fixes something else - the above question is more of a clarifying question.